### PR TITLE
Create pull request template

### PR DESCRIPTION
**Pull request by OfirMatasas**
This pull request template will help us provide all the information needed to help the reviewer to understand what changes we're trying to insert to our project, and to make sure that all scenarios are tested.

Each pull request we'll open in the future, using this template, is going to look like this:
image

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+### Background context of the changes:
+Why do we need this changes?
+
+### The ticket this PR closes:
+Put a link to the ticket on Jira we're closing using this pull request.
+
+### Where should the reviewer start?
+If the changes are spreading over serveral files, let the reviewer know where to start.
+
+### Scenarios that were tested during this imolementation:
```
Comment (LinMoskovitch) [inline .github/pull_request_template.md:18]: typo in `imolementation`

**Review by LinMoskovitch — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (OfirMatasas) [reply .github/pull_request_template.md]: Fixed

**Review by LinMoskovitch — APPROVED** [review]
LGTM

**Review by DekelR — APPROVED** [review]
It looks good - nothing to add besides holding a list of all current features that are working to guarantee that the new changes didn't (by mistake) harm some other, And after the PR war merged, add it to the list of things that need to be checked.

**Pull request merged**